src/main/java/uk/gov/ons/ctp/integration/mock/endpoints/HelpEndpoint.java

```diff
@@ -73,9 +69,11 @@
     helpText.append("  with the newly capture data.\n");
     for (uk.gov.ons.ctp.integration.mock.endpoints.RequestType requestType :
         uk.gov.ons.ctp.integration.mock.endpoints.RequestType.values()) {
-      helpText.append("\n");
-      describeUrl(helpText, requestType, "/capture");
-      describeQueryParams(helpText, requestType);
+      if (!requestType.name().contains("CASE")) {
+        helpText.append("\n");
+        describeUrl(helpText, requestType, "/capture");
+        describeQueryParams(helpText, requestType);
+      }
     }
 
     helpText.append("\n\n");
```

Comment:
line 86
```
curl -s localhost:8162/addresses/partial?input=Treganna?offset=625;limit=85
```
after Treganna should be ampersand I think, and perhaps the semicolon before limit should ampersand?

Reply:
Yes, should follow standard url query parameter format, so '?' in front of first param and '&' in front of subsequent query params

Reply:
done